packages/lodestar/src/chain/blocks/importBlock.ts

```diff
@@ -284,8 +285,9 @@ async function maybeIssueNextProposerEngineFcU(
       const proposerIndex = prepareState.epochCtx.getBeaconProposer(prepareSlot);
       const feeRecipient = chain.beaconProposerCache.get(proposerIndex);
       if (feeRecipient) {
+        const safeBlockHash = chain.forkChoice.getJustifiedBlock().executionPayloadBlockHash ?? ZERO_HASH_HEX;
```

Comment:
I prefer to have a separate method like this in forkchoice
```python
def get_safe_execution_payload_hash(store: Store) -> Hash32:
    safe_block_root = get_safe_beacon_block_root(store)
    safe_block = store.blocks[safe_block_root]

    # Return Hash32() if no payload is yet justified
    if compute_epoch_at_slot(safe_block.slot) >= BELLATRIX_FORK_EPOCH:
        return safe_block.body.execution_payload.block_hash
    else:
        return Hash32()
```

and use it everywhere to make it clear

Reply:
if you think `safeBlockHash()` will not changed in the future then we're good 👍